script/update_eligibility.rb

```diff
@@ -97,11 +97,15 @@ def check_and_run
       active_thh = active_household.latest_active_thh_with_year(effective_date.year)
       deter = active_thh.try(:latest_eligibility_determination)
 
-      if active_thh && deter.present? && deter.csr_percent_as_integer.to_s == csr && active_thh.effective_starting_on.to_date == effective_date && deter.max_aptc.to_f == aptc.to_f
+      if active_thh && deter.present? && active_thh.effective_starting_on.to_date == effective_date && deter.max_aptc.to_f == aptc.to_f
+        csr_int = (csr == 'limited' ? '-1' : csr)
+        active_thh.tax_household_members.each do |thm|
```

Comment:
Not supposed to be looping here, we should individually update the CSR for each member from the provided CSV. Should use  person_hbx_id to find the member to update the csr.